uiowa/uiowa
# Graduate Admissions Migrate

Resolves https://github.com/uiowa/uiowa/issues/4638
Resolves #4740 

## To Test
get admissions_grad in the Grad docroot pulled and resolving in DevDesktop
blt ds --site grad.admissions.uiowa.edu
drush @admissionsgrad.local en grad_admissions_migrate
drush @admissionsgrad.local uli admin/config/sitenow/sitenow-migrate
Enter details for migration using DevDesktop settings indicated in help text. Database name is 'admissions_grad'.
**delete existing [AoS content](https://admissionsgrad.uiowa.ddev.site/admin/content?type=area_of_study), [Aos taxonomy](https://admissionsgrad.uiowa.ddev.site/admin/structure/taxonomy/manage/grad_areas_of_study/overview), [Colleges taxonomy](https://admissionsgrad.uiowa.ddev.site/admin/structure/taxonomy/manage/colleges/overview), [Degree types taxonomy](https://admissionsgrad.uiowa.ddev.site/admin/structure/taxonomy/manage/degree_types/overview) data**
drush @admissionsgrad.local ms to see migration information
drush @admissionsgrad.local mim --all to run all migrations

- there will be broken links output as part of this. we are waiting for a map of replacements from Admissions.
- check pieces of content for alignment and accuracy
- colleges taxonomy should combine description and program info from the old site.
- estimated costs should map up to replacement domestic version
- department and catalog links should be combined into related links
